File: datacollector/app/entities/bolus.go
package entity

type Bolus struct {
	ID           uint   `json:"id"`
	Number       string `json:"number"`
	Version      string `json:"version"`
	BatchNumber  string `json:"batch_number"`
	DeviceNumber string `json:"device_number"`
	ProducedAt   string `json:"produced_at"`
}

func (Bolus) TableName() string {
	return "boluses"
}

File: app/Models/Bolus.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Bolus extends Model
{
    use HasFactory;
    protected $fillable = [
        'number',
        'version',
        'batch_number',
        'produced_at',
        'is_active'
    ];

    protected $casts = [
        'is_active' => 'boolean'
    ];

    protected $table='boluses';
}
